packages/celotool/src/lib/geth.ts

```diff
@@ -859,6 +859,10 @@ export async function startGeth(
   const privateKey = instance.privateKey || ''
   const lightserv = instance.lightserv || false
   const etherbase = instance.etherbase || ''
+  // Allow maxPeers to be zero
+  const maxPeers = instance.maxPeers !== undefined ? instance.maxPeers : gethConfig.instances.length
```

Comment:
Why default to `gethConfig.instances.length`? That seems low and it would be better, in my opinion to default to not providing the flag.